### PR TITLE
REST: Allow syncing of maps over WAN

### PR DESCRIPTION
**Pull request by emrahkocaman**
Allow initialization from REST API

### CODE REVIEW

**Comment by emrahkocaman** [comment]
run-lab-run

**Comment by pveentjer** [comment]
Can you modify the title of this ticket so it becomes more descriptive?

File: hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java [hunk 1]
```diff
@@ -283,4 +283,11 @@ public void testKillNodeWithWrongCredentials() throws IOException {
 
         assertEquals("{\"status\":\"forbidden\"}", communicator.killMember("dev1", "dev-pass"));
     }
+
```
Comment (bilalyasar) [inline hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java]: missing `test annotation`

**Comment by muratayan** [comment]
👍

File: hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java (hunk 1 above)
Comment (emrahkocaman) [inline hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java]: 👍

**Comment by emrahkocaman** [comment]
run-lab-run

**Comment by serkan-ozal** [comment]
:+1:

**Comment by bilalyasar** [comment]
👍

**Pull request merged**